openedx/features/enterprise_support/api.py

```diff
@@ -233,6 +234,32 @@ def enterprise_enabled():
     return 'enterprise' in settings.INSTALLED_APPS and getattr(settings, 'ENABLE_ENTERPRISE_INTEGRATION', True)
 
 
+def enterprise_customer_for_request(request, tpa_hint=None):
```

Comment:
Not sure what you mean by "go all the way out to edx-enterprise". Why can't the abstraction live in edx-enterprise if it is enterprise-specific? I think it makes sense to put enterprise-related code that needs to use code from edx-platform into this module, but it doesn't seem like that is the case with this function.